src/coreclr/jit/unwindamd64.cpp

```diff
@@ -119,6 +119,78 @@ short Compiler::mapRegNumToDwarfReg(regNumber reg)
         case REG_XMM15:
             dwarfReg = 32;
             break;
+        case REG_XMM16:
```

Comment:
This function should only ever see "Preserved across function calls = true" registers. None of this should be needed.

Reply:
Just to confirm, all of the XMM and K register handling here can be removed since they are all "volatile" and considered "callee save"?

Reply:
I think so.

Reply:
Just double checked and looks like this is called from two places:
1. `unwindPushPopCFI`
2. `unwindSetFrameRegCFI`

The former looks to be only called from `unwindPushPopMaskCFI` which is itself only called on arm.

The latter is called from `unwindSetFrameReg` but looks to only ever be `REG_FPBASE` which is ESP or EBP

So does indeed look to be fine to remove.